pages/guideforms.py

```diff
@@ -560,14 +560,14 @@ def simple_html_output(self):
                 row_template = checkbox_row
                 if field.widget.check_test(value):
                     checked = True
-                
+
```

Comment:
I'm pretty sure these phantom spacings are from me - I'll try and be more diligent about avoiding this!

Reply:
I know I have left a few.  VSCode wasn't cleaning up automatically as I thought it ought to have been.